hack/binary_deps.py

```diff
@@ -9,16 +10,25 @@
 import tempfile
 import subprocess
 import shlex
-import os
 
 log = logging.getLogger(__name__)
 
+# assume linux
 KUBEBUILDER_URL = "https://github.com/kubernetes-sigs/kubebuilder/releases/download/v2.3.1/kubebuilder_2.3.1_linux_amd64.tar.gz"
 KUSTOMIZE_URL = "https://github.com/kubernetes-sigs/kustomize/releases/download/v3.1.0/kustomize_3.1.0_linux_amd64"
-GO_URL = "https://dl.google.com/go/go1.13.12.linux-amd64.tar.gz"
+GO_URL = "https://dl.google.com/go/go1.13.15.linux-amd64.tar.gz"
```

Comment:
Would be good to extract the version info from here. I Manager we have a `.go-version` file 

```
 michal@lh:scylladb/mermaid (mmt/2064-estimate-parallelism)$ cat .go-version 
1.13.15
```

Reply:
There are at least 3 places where we have the go version:

1. `go.mod`
2. `Dockerfile`
3. `hacks/binary_deps.py`

I can't think of an easy solution to get all three from one source.

My current best solution is to do the `.go-version` file and add documentation for go version. Can you think of something else?